script/postinstall.js

```diff
@@ -7,6 +7,6 @@ if (process.platform.indexOf('win') === 0) {
 } else {
   script += '.sh'
 }
-var child = cp.exec(script, [], {stdio: ['pipe', 'pipe', 'pipe']})
+var child = cp.spawn(script, [], {stdio: ['pipe', 'pipe', 'pipe']})
```

Comment:
This change won't work as expected on Windows. You need to add `shell: true` to the options object;
```js
var child = cp.spawn(script, [], { stdio: ['pipe', 'pipe', 'pipe'], shell: true })
```

---
_Ref: https://nodejs.org/dist/latest-v7.x/docs/api/child_process.html#child_process_spawning_bat_and_cmd_files_on_windows_